src/poke_env/environment/double_battle.py

```diff
@@ -337,36 +338,36 @@ def available_switches(self) -> List[List[Pokemon]]:
         return self._available_switches
 
     @property
-    def can_dynamax(self) -> List[bool]:
+    def can_mega_evolve(self) -> List[bool]:
         """
-        :return: Whether or not the current active pokemon can dynamax
+        :return: Whether or not either current active pokemon can mega evolve.
         :rtype: List[bool]
         """
-        return self._can_dynamax
+        return self._can_mega_evolve
 
     @property
-    def can_mega_evolve(self) -> List[bool]:
+    def can_z_move(self) -> List[bool]:
         """
-        :return: Whether or not either current active pokemon can mega evolve.
+        :return: Whether or not the current active pokemon can z-move.
         :rtype: List[bool]
         """
-        return self._can_mega_evolve
+        return self._can_z_move
 
     @property
-    def can_tera(self) -> List[Union[bool, PokemonType]]:
+    def can_dynamax(self) -> List[bool]:
         """
-        :return: Whether or not the current active pokemon can terastallize. If yes, will be a PokemonType.
-        :rtype: List[Union[bool, PokemonType]]
+        :return: Whether or not the current active pokemon can dynamax
+        :rtype: List[bool]
         """
-        return self._can_tera
+        return self._can_dynamax
 
     @property
-    def can_z_move(self) -> List[bool]:
+    def can_terastallize(self) -> List[Union[bool, PokemonType]]:
```

Comment:
Why go from `can_tera` to `can_terastallize`? Should we changed the class attribute name as well?

Reply:
Good point, I think I just changed this in the fray of a bunch of changes. Changing it back now: 95a61e802ea887f374d86d770a7d18110c6a6470